### DISCUSSION TITLE
DFL for different filter settings

### DISCUSSION THREAD

**Discussion by PCBenetton** [gdyuldin/x6100_gui#36] [Ideas]
Add different filter settings saved by mode (maybe 3 for each mode is enough) that cycle with DFL button or a Filter menu on DFL button and filter # may be added to MFK knob. Much like the factory FW. 
The cycling with DFL is much more useful, the configuration may be done on the filter page (VOL) of the GEN button selections.
Useful for every mode but particularly so for CW (IE switch between 500Hz and 50Hz easily).

Thanks for the awesome work!!
73 DE PP2PB

**Comment by gdyuldin** [comment]
I'll think about how to perform it.
